bundle/config/mutator/process_target_mode.go

```diff
@@ -105,6 +106,15 @@ func transformDevelopmentMode(ctx context.Context, b *bundle.Bundle) diag.Diagno
 		// (registered models in Unity Catalog don't yet support tags)
 	}
 
+	for i := range r.QualityMonitors {
```

Comment:
just to confirm: this does not mess with the production version of these monitors, which should still have schedules intact?

Reply:
Yeah, indeed, `transformDevelopmentMode()` is only applied when a deployment target uses `mode: development`.